### DISCUSSION TITLE
LoadCircle没有单独开线程，造成阻塞，如何开启多线程避免这种情况？

### DISCUSSION THREAD

**Discussion by 11D-Beyonder** [HandyOrg/HandyControl#1024] [Q&A] [answered]
点击按钮后，LoadCircle不再转动。
![LoadCircle Question.gif](https://i.loli.net/2021/10/20/guUX638M5icaCbH.gif)
Button的响应如下，用循环模拟处理过程。
```csharp
private void SaveFile(object sender, RoutedEventArgs e)
{
	if (file == null)
	{
		Growl.Error("文件不存在");
	}
	else
	{
		for (int i = 1; i <= 2000000000; i++) ;
	}
}
```
想要实现的功能是：点击按钮前LoadCircle不显示，点击后LoadCircle转动，处理结束后LoadCircle不显示。

**Comment by NaBian** [comment, marked as answer]
这个和 ui 库没有关系，需要你自己使用异步。

**Reply by 11D-Beyonder** [reply]
那请问如何开启跨线程的UI，使得加载界面时LoadingCircle转动？

**Reply by GF-Huang** [reply]
正正经经找本 WPF 或者 WinForm 的书看学习下再说，这个问题属于非常基础的常识问题。
